assets/react/v3/entries/course-builder/components/layouts/Notebook.tsx

```diff
@@ -0,0 +1,366 @@
+				wrapper.style.top = top === 'auto' ? `${window.innerHeight - notebook.MIN_NOTEBOOK_HEIGHT}px` : top;
+				if (isDefined(height)) {
+					wrapper.style.height = height;
+				}
+				if (isDefined(width)) {
+					wrapper.style.width = width;
+				}
+			}
+		}
+	}, [isCollapsed, isFloating]);
+
+	useEffect(() => {
+		const notebookData: NotebookData = JSON.parse(getFromLocalStorage(LocalStorageKeys.notebook) || '{}');
+		if (notebookData.notes) {
+			setContent(notebookData.notes);
+		}
+	}, []);
+
+	return (
+		<animated.div ref={wrapperRef} css={styles.wrapper({ isCollapsed, isFloating })} style={{ ...expandAnimation }}>
+			<div css={styles.header({ isCollapsed })} onMouseDown={handleMouseDown}>
+				<span css={styleUtils.text.ellipsis(1)}>Notebook</span>
+				<div css={styles.actions}>
+					<Button
+						variant="text"
+						size="small"
+						onClick={() => {
+							setIsCollapsed(!isCollapsed);
+							setIsFloating(false);
+						}}
+						buttonCss={styles.collapseButton({ isCollapsed })}
+					>
+						<SVGIcon name="plusMinus" height={24} width={24} />
+					</Button>
+					<Button
+						variant="text"
+						size="small"
+						onClick={() => {
+							setIsCollapsed((previouState) => {
+								if (isFloating) {
+									return true;
+								}
+								if (previouState) {
+									return false;
+								}
+
+								return previouState;
+							});
+							setIsFloating(!isFloating);
+						}}
+					>
+						<SVGIcon name="arrowsIn" height={24} width={24} />
+					</Button>
+					<Show when={isFloating}>
+						<Button
+							variant="text"
+							size="small"
+							onClick={() => {
+								setIsFloating(false);
+								setIsCollapsed(true);
+							}}
+						>
+							<SVGIcon name="cross" height={24} width={24} />
+						</Button>
+					</Show>
+				</div>
+			</div>
+			<div
+				css={styles.notebook}
+				contentEditable={contentEditable}
+				onBlur={(event) => onContentBlur(event)}
+				onClick={() => setContentEditable(true)}
+				onKeyDown={(event) => {
+					if (event.key === 'Escape') {
+						event.preventDefault();
+						event.currentTarget.blur();
+					}
+				}}
+				dangerouslySetInnerHTML={{ __html: content }}
+			/>
+
+			<Show when={isFloating && !isCollapsed}>
+				<button
+					type="button"
+					css={[styleUtils.resetButton, styles.textFieldExpand]}
+					onMouseDown={(event) => handleResize(event, 'bottom-right')}
+				>
+					<SVGIcon name="textFieldExpand" height={16} width={16} />
+				</button>
+			</Show>
+		</animated.div>
+	);
+};
+
+export default Notebook;
+
+const styles = {
+	wrapper: ({
+		isCollapsed,
+		isFloating,
+	}: {
+		isCollapsed: boolean;
+		isFloating: boolean;
+	}) => css`
+		position: fixed;
+		background-color: ${colorTokens.background.active};
+		bottom: 0;
+		right: 0;
+		width: 360px;
+		border-radius: ${borderRadius.card} 0 ${borderRadius.card} 0;
+		transition: box-shadow background 0.3s ease-in-out;
+		box-shadow: 0 0 4px 0 rgba(0, 30, 43, 0.16);
```

Comment:
Add this box shadow to the `styles.ts` file. The shadow name could be `notebook`